src/interfaces/communication/ruuvi_interface_communication_ble_advertising.h

```diff
@@ -43,6 +43,23 @@ typedef struct
     int8_t rssi;      //!< RSSI of advertisement
     uint8_t data[RUUVI_COMM_BLE_ADV_SCAN_LENGTH]; //!< Full payload of the advertisement
     size_t data_len;  //!< Length of received data
+    bool is_coded_phy; //!< True if Coded PHY was used
+    uint8_t primary_phy; /**< Indicates the PHY on which the primary advertising
```

Comment:
Using uint instead of some named enum is a good choice here, if we ever want to support other platforms such as some mesh platform we can make a compatibility layer. No reason to put effort Into writing `ri_communication_primary_phy` enum for now.